src/command_modules/azure-cli-network/azure/cli/command_modules/network/_template_builder.py

```diff
@@ -219,34 +219,37 @@ def build_load_balancer_resource(name, location, tags, backend_pool_name, fronte
         ],
         'frontendIPConfigurations': [frontend_ip_config]
     }
-
     lb = {
         'type': 'Microsoft.Network/loadBalancers',
         'name': name,
         'location': location,
         'tags': tags,
-        'apiVersion': '2015-06-15',
+        'apiVersion': get_api_version(ResourceType.MGMT_NETWORK),
         'dependsOn': [],
         'properties': lb_properties
     }
+    if sku and supported_api_version(ResourceType.MGMT_NETWORK, min_api='2017-08-01'):
```

Comment:
I agree from a technical perspective, but my concern is that if I don't make it clear *why* this is not just part of the properties dictionary, then a dev in the future might see this as a place to simplify the code and put it in the dictionary. This would then break older profiles.

For that reason, I would like to keep it explicit.